### ISSUE TITLE
Cannot Remove Account

### ISSUE THREAD

**Issue by allanrivera** [awslabs/workload-discovery-on-aws#315]
If your issue relates to the _Discovery Process_, please first follow the steps described in the implementation guide [Debugging the Discovery Component](https://docs.aws.amazon.com/solutions/latest/aws-perspective/appendix-e-debugging-the-discovery-component.html)

---

**Describe the bug**
Attempting to remove an account or region hangs on the 'Remove Account' popup and produces the following error:
```
"o
2 validation errors detected: Value '[]' at 'accountAggregationSources.1.member.awsRegions' failed to satisfy constraint: Member must have length greater than or equal to 1; Value '[]' at 'accountAggregationSources.1.member.accountIds' failed to satisfy constraint: Member must have length greater than or equal to 1"
```

**To Reproduce**

1. Go to '/accounts'
2. Click on 'Remove'
3. Click 'Delete'

**Expected behavior**
The account is removed

**Screenshots**
![image](https://user-images.githubusercontent.com/100428906/200631589-2713263e-a60d-48cc-919c-ca464b66d444.png)

**Browser (please complete the following information):**

- Name chrome, safari
- Version 107.0.5304.87, 16.0

**Additional context**

**Comment by svozza** [comment]
Does this happen for any account you try to remove? Does the account you tried to remove still exist in the Settings DynamoDB table? The table will have a long name like this: `<your-WD-stack-name>-SettingsResolversStack-<ID-string>-PerspectiveSettingsTable-<ID-string>`.

**Comment by allanrivera** [comment]
The account exists in the DDB table.  I do not have another account to test this with.

**Comment by svozza** [comment]
Yeah, I've bee able to reproduce this. What's happening is that if you've only one account and try to remove it we're sending an empty array to AWS Config for the number of regions to to scan, hence the validation error. As a workaround, you can just delete the account directly from the DynamoDb table.

**Comment by allanrivera** [comment]
Thanks!

**Comment by svozza** [comment]
I'm going to keep this issue open because I need to fix it on my end. I will close it when we've shipped the fix.

**Comment by svozza** [comment]
This issue has been fixed in the new v2.0.1 release. You can download the updated template here: https://aws.amazon.com/solutions/implementations/workload-discovery-on-aws/.
